disnake/gateway.py

```diff
@@ -424,6 +425,7 @@ async def from_client(
         ws._discord_parsers = client._connection.parsers
         ws._dispatch = client.dispatch
         ws.gateway = gateway
+        ws.resume_gateway = gateway
```

Comment:
does this not have the potential to be the resume gateway?

Reply:
Yeah, on initial connect it'll be the default gateway until it receives the `READY` and overwrites it.
Shouldn't matter, as we won't resume before receiving a `READY`, at which point it gets overwritten with the "true" resume url.